src/app/node-data/template.html

```diff
@@ -47,21 +47,55 @@
       </mat-error>
     </mat-form-field>
 
-    <mat-form-field>
+    <mat-checkbox [formControlName]="Controls.UseAutoscaling"
+                  class="km-use-autoscaling"
+                  fxFlex>
+      Use horizontal node autoscaling
+    </mat-checkbox>
+
+    <mat-form-field *ngIf="!form.get(Controls.UseAutoscaling).value>
       <mat-label>Replicas</mat-label>
-      <input [formControlName]="Controls.Count"
-             id="km-node-count-input"
-             matInput
-             type="number"
-             min="0"
-             autocomplete="off"
-             required>
+      <input [formControlName]="
+                    Controls.Count"
+                    id="km-node-count-input"
+                    matInput
+                    type="number"
+                    min="0"
+                    autocomplete="off"
+                    required>
       <mat-error *ngIf="form.get(Controls.Count).hasError('required')">Number of replicas is required.</mat-error>
       <mat-error *ngIf="form.get(Controls.Count).hasError('ipsMissing')">
         Not enough IPs left. Reduce number of replicas or add more machine networks.
       </mat-error>
     </mat-form-field>
 
+    <mat-form-field fxFlex
+                    *ngIf="form.get(Controls.UseAutoscaling).value">
+      <mat-label>Min Replicas*</mat-label>
+      <input [formControlName]="Controls.MinReplicas"
+             id="km-node-min-replicas-input"
+             matInput
+             type="number"
+             min="0"
+             autocomplete="off">
+    </mat-form-field>
+
+    <mat-form-field fxFlex
+                    *ngIf="form.get(Controls.UseAutoscaling).value">
+      <mat-label>Max Replicas*</mat-label>
```

Comment:
`*` should be coming from `required` directive. Is it intended, if yes then why?